src/adapter.js

```diff
@@ -3,92 +3,102 @@
 const { filter, sortAll, take, map } = require('./utils')
 const drain = require('it-drain')
 
-class InterfaceDatastoreAdapter {
-  async open () { // eslint-disable-line require-await
-
+/**
+ * @typedef {import('./key')} Key
+ * @typedef {import('./types').Pair} Pair
+ * @typedef {import('./types').Datastore} Datastore
+ * @typedef {import('./types').Options} Options
+ * @typedef {import('./types').Query} Query
+ * @typedef {import('./types').Batch} Batch
+ */
+
+/**
+ * @template O
+ * @typedef {import('./types').AwaitIterable<O>} AwaitIterable
+ */
+
+/**
+ * @implements {Datastore}
+ */
+class Adapter {
```

Comment:
Why does this class not appear in the [generated docs](https://ipfs.github.io/interface-datastore/#has)?

Reply:
thats the old docs, the new docs should show this class.